### ISSUE TITLE
Build errors

### ISSUE THREAD

**Issue by saskwatch54** [pazaan/600SeriesAndroidUploader#164]
2 error(s), 0 warning(s)
--
Build - 2 error(s), 0 warning(s)
Error: No matching   files were found with search pattern: d:\a\1\s\**\*.sln
Packages failed to   install

**Comment by saskwatch54** [comment]
2 error(s), 0 warning(s)
--
Build - 2 error(s), 0 warning(s)
Error: No matching   files were found with search pattern: d:\a\1\s\**\*.sln
Packages failed to   install

**Comment by pazaan** [comment]
Can we get some more details about this? Which branch were you trying to build? Which version of Android Studio are you running?

**Comment by saskwatch54** [comment]
Thanks for getting back to me. I made an error and had merged forks, incorrectly. I deleted the merged fork, redid the fork and it worked. 
Thank you for doing this. We are seeing my sons numbers at night. Can’t thank you enough. 
Pat Hughes 


Sent from Yahoo Mail for iPhone


On Monday, January 8, 2018, 8:20 PM, Lennart Goedhart <notifications@github.com> wrote:


Can we get some more details about this? Which branch were you trying to build? Which version of Android Studio are you running?
